src/getCoinCombination.test.js

```diff
@@ -7,4 +7,44 @@ describe('getCoinCombination', () => {
     expect(getCoinCombination)
       .toBeInstanceOf(Function);
   });
+
+  it('should return [0, 0, 0, 0] for 0 ', () => {
+    expect(getCoinCombination(0)).toEqual([0, 0, 0, 0]);
+  });
+
+  it('If accept pennys only', () => {
+    expect(getCoinCombination(1)).toEqual([1, 0, 0, 0]);
+  });
+
+  it('If accept only nikel', () => {
+    expect(getCoinCombination(5)).toEqual([0, 1, 0, 0]);
+  });
+
+  it('If accept only dime', () => {
+    expect(getCoinCombination(10)).toEqual([0, 0, 1, 0]);
+  });
+
+  it('If accept  only quoters ', () => {
```

Comment:
The test description 'If accept only quoters' should be revised for clarity and correct spelling. Consider changing it to 'should return [0, 0, 0, 1] for 25 cents (only quarters)'. The correct spelling is 'quarters'.